verify-merge.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+import glob
+
+GIT = os.getenv('GIT','git')
+GPG = os.getenv('GPG','gpg')
+
+def verify():
+    global args, workdir
+    os.chdir('gitian-pubkeys')
+    print('Importing pubkeys...')
+    keys = [f for f in glob.glob("*.asc", recursive=True)]
+    for key in keys:
+        subprocess.check_call([GPG, '--import', key])
+    print('Refreshing pubkeys...')
+    subprocess.check_call([GPG, '--refresh'])
```

Comment:
I don't think this is a good idea to just do automatically... the user should determine if / when their keys are refreshed (especially considering this is [now a known attack vector](https://gist.github.com/rjhansen/67ab921ffb4084c865b3618d6955275f)).  In my case `refreshing 486 keys from hkps://keys2.kfwebs.net` was not desired.

Maybe just refresh the 1 supplied key if you feel that is necessary.